src/core/aws-utils.ads

```diff
@@ -80,6 +80,13 @@ package AWS.Utils is
    --  General helper functions --
    -------------------------------
 
+   procedure Set_Random_Task_Safe (Flag : Boolean := True);
```

Comment:
In fact I would just remove this and have a unique and safe mode. As discussed the random generator is not called a lot and not in critical part of AWS.

Reply:
Yes, it is protected always now.